README.md

```diff
@@ -123,18 +122,25 @@ Some mathematical set relations will be useful throughout this specification.
 We shall denote:
 
 * The equivalence of two sets `A` and `B` as `A = B`
-* The intersection as `A ∩ B`
-* The union as `A ∪ B`
-* The set difference as `A \ B`
-* The symmetric difference as `A △ B`
-* The subset relation as `A ⊆ B`
-* The proper subset relation as `A ⊂ B`
+* The [subset] relation as `A ⊆ B` ("A is a subset of B")
+* The [proper subset] relation as `A ⊂ B` ("A is a subset of B, AND A is not B")
+* The [intersection] as `A ∩ B`
+* The [union] as `A ∪ B`
+* The [set difference] as `A \ B`
+* The [symmetric difference] as `A △ B`
```

Comment:
Nice to have these links. Can we aim to have *all* links in the "Informative references" section?

Reply:
done